hbase-server/src/main/java/org/apache/hadoop/hbase/ipc/RpcCall.java

```diff
@@ -83,7 +83,7 @@ public interface RpcCall extends RpcCallContext {
   /** Returns The request header of this call. */
   RequestHeader getHeader();
 
-  ConnectionHeader getConnectionHeader();
```

Comment:
While we are here, I wonder if we should retain parity with request attributes. We could add a `Map<String, byte[]> getRequestAttributes()`. I think the impl for that should be lazy since there's no point spending cpu/memory creating the map if it's not going to be used. You can use this in your slow logs PR, or you could add it in that PR if you want. I just want to try to see if we can get it in before 2.6.0 drops.

Please add javadoc to this new method